solutions/day_5/solution.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+from functools import reduce
+
+
+TEXT_FILE_LOCATION = "input.txt"
+BATCH_SIZE = 3
+
+CRATE_HOLDER = {1: [], 2: [], 3: [], 4: [], 5: [], 6: [], 7: [], 8: [], 9: []}
```

Comment:
I wouldn't hard code the size of this, but make it initialised based on the input file.